archive/overlap.go

```diff
@@ -198,3 +209,199 @@ func mergeLargestChunkSpanInfos(spans []SpanInfo, order zbuf.Order) []SpanInfo {
 	}
 	return append(res, mergeSpanInfos(run, order))
 }
+
+// removeMaskedChunks performs an in-place edit of the spans input slice,
+// updating each SpanInfo to remove chunks that mask other chunks within the
```

Comment:
By "masked" you mean a chunk completely contained within the span of one other chunk , right?

Reply:
That's correct; I used that term in the design writeup in #1183 . I can add a comment in the code defining that usage; I think it'd go best with the `Masks` field of the `Chunk` struct, what do you think?

Reply:
I added a comment describing masks here: 50db6ed898636685b65ee25ffcde78436a96236f